api/services/BedCountService.js

```diff
@@ -62,7 +62,7 @@ const getReinstatementTester = (checkoutEventsRangeFactory) => (reinstatement, e
   const personIdMatches = reinstatement.person_id === event.person_id;
   const timestampMatches = checkoutEventsRangeFactory(reinstatement.timestamp).contains(event.timestamp);
 
-  return operationMatches && personIdMatches && timestampMatches && { reinstatement, event };
+  return operationMatches && personIdMatches && timestampMatches && {reinstatement, event};
```

Comment:
Do you want to define this as a code style? I'll need to reconfigure my IDE if you're going to keep removing these spaces.